gode/packages.go

```diff
@@ -67,8 +67,9 @@ func RemovePackages(packages ...string) error {
 // OutdatedPackages returns a map of packages and their latest version
 func OutdatedPackages(names ...string) (map[string]string, error) {
 	args := append([]string{"outdated", "--json"}, names...)
-	stdout, stderr, err := execNpm(args...)
-	if err != nil {
+	stdout, stderr, _ := execNpm(args...)
+	// Check stderr since npm outdated returns exit code 1 when there are outdated packages
+	if len(stderr) > 0 {
```

Comment:
being _totally_ paranoid, would it be safer to check both the exit code AND stderr length? iow, if we have an exit code > 0 AND a message output via stderr, then it's an error.